chain/ledgerStore.go

```diff
@@ -20,6 +20,7 @@ type ILedgerStore interface {
 	GetHeaderByHeight(height uint32) (*block.Header, error)
 	GetTransaction(hash Uint256) (*transaction.Transaction, error)
 	GetName(registrant []byte) (string, error)
+	GetRegName(name string) (string, error)
```

Comment:
getregname typically means get name from something else. What we are doing is actually GetRegistrant. We can reuse the GetRegistrant method to be compatible to current sdk.